src/xtgeo/grid3d/_gridprop_import_eclrun.py

```diff
@@ -475,7 +475,7 @@ def section_generator(generator):
 def import_gridprops_from_restart_file_sections(
     sections,
     names: Union[List[str], "str"],
-    dates: Union[int, str],
+    dates: Union[List[int], str, List[str]],
```

Comment:
Pure `str` should not be there, use `Literal["all", "last", "first"]` instead?

Reply:
Literal is perhaps problematic at the moment because of py 3.6?